### ISSUE TITLE
Ionic 4 tabs starter unusable. While scrolling tab navigation is blocked.

### ISSUE THREAD

**Issue by d3vhound** [ionic-team/ionic#16863]
# Bug Report

**Ionic version:**
[x] **4.x**

**Current behavior:**
When scrolling and pressing tab buttons, navigation is blocked. Only on native devices/emulators.

**Expected behavior:**
I've been testing ionic 3 apps and this problem doesn't exist, only on ionic 4 with capacitor and cordova. Is this intentional? If so is there a way to disable this? Having to double click the tab icon to navigate each time you have just scrolled content is annoying and unusable.

**Steps to reproduce:**
Create tab start project. Add capacitor/cordova. Run on ios/android emulator and while scrolling (important) click on a tab. The content must be in a scrolling state. If the content is still then it will work normally. This issue is more evident with a long list of items as the content takes longer to come to a standstill. 

**Ionic info:** 

```
Ionic:

   ionic (Ionic CLI)             : 4.6.0 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.0-rc.0
   @angular-devkit/build-angular : 0.11.4
   @angular-devkit/schematics    : 7.1.4
   @angular/cli                  : 7.1.4
   @ionic/angular-toolkit        : 1.2.2

Capacitor:

   capacitor (Capacitor CLI) : 1.0.0-beta.13
   @capacitor/core           : 1.0.0-beta.13

System:

   NodeJS : v11.4.0 (/usr/local/bin/node)
   npm    : 6.4.1
   OS     : macOS Mojave
```

**Comment by d3vhound** [comment]
This is definitively an ionic 4 issue. The Ionic 3 starter tabs project works fine with capacitor/cordova.

**Comment by hiepxanh** [comment]
yeah, I  have the same problem, when I have a list of item and click on it will navigate to next page. I can see the different in the behavior is my sumary:

###Native App: 
Action 1: Right now if you "grab" a scroll view to stop the inertial scrolling after throwing it, that will stop the scroll.
Action 2: If you tap very quick (<300ms) it will move to new page.

###Ionic 3 version: 
Action 1: Right now if you "grab" a scroll view to stop the inertial scrolling after throwing it, that will trigger a click immediatly. That could be so annoying. 

Opinion 1: It was fixed  for Ionic 1 in [#2665](https://github.com/ionic-team/ionic/issues/2665). But Ionic 3 is not, but we can use solution in Ionic 1 [#1438](https://github.com/ionic-team/ionic/issues/1438).
Opinion 2: we can click other tab while it scrolling.

###Ionic 4 version: 
Action 1: Right now if you "grab" a scroll view to stop the inertial scrolling after throwing it,  that will stop the scroll.
Action 2: If you tap very quick (<300ms) it will NOT move, nor trigger any click.

Opinion 1: I don't know what stop my ion-item to recognize click why scrolling. can someone show me which one. So I can find solution
Opinion 2: we CAN'T click other tab while it scrolling.

My UX design said. Ionic 4 have great performance and smooth almost like native app. Just only this problem cause some annoy make people few that it lagging.

**Comment by BastienAlvez** [comment]
It seems that all click events (I did not test the others) are disabled when scrolling not only on tab navigation
